File: cpp/bc1134.cpp
#include <iostream>

using namespace std;

int main()
{
	int qAl, qGas, qDi, val;
	
	qAl = qGas = qDi = 0;
	
	while(true)
	{
		cin >> val;
		
		if(val == 4) break;
		else
		{
			switch(val)
			{
				case 1: qAl++; break;
				case 2: qGas++; break;
				case 3: qDi++; break;
			}
		}
	}
	
	cout << "MUITO OBRIGADO" << endl;
	cout << "Alcool: " << qAl << "\nGasolina: " << qGas << "\nDiesel: " << qDi << endl;
}

File: python/bc1134.py
al = 0
gas = 0
di = 0

while True:
    tCo = int(input())

    if tCo == 1: al += 1
    elif tCo == 2: gas += 1
    elif tCo == 3: di += 1
    else:
        if tCo == 4: break

print("MUITO OBRIGADO")
print(f"Alcool: {al}\nGasolina: {gas}\nDiesel: {di}")